src/base-element.js

```diff
@@ -202,7 +202,7 @@ export class BaseElement {
    * Intended to be implemented by subclasses. Tests whether the element
    * requires fixed positioning.
    * @return {boolean}
-   * @protected
+   * @public
```

Comment:
What just easier to quickly replace with intact formatting. I don't really understand what the problem here was. Will need to investigate. The usages should all have been legal.